p2p/config/config.go

```diff
@@ -56,6 +56,7 @@ type Config struct {
 	SwarmConfig           SwarmConfig   `mapstructure:"swarm"`
 	BufferSize            int           `mapstructure:"buffer-size"`
 	MsgSizeLimit          int           `mapstructure:"msg-size-limit"` // in bytes
+	LocalNetwork          bool          `mapstructure:"local-network"`
```

Comment:
this needs to be renamed to something less specific. e.g. RandConnIncludeOutbound

Reply:
There is another point of view: The only responsibility of this configuration option is to start local network. It also means you must not use it on normal network. What you offer here does not have any responsibity meaning and just makes configuration more complicated. By the way it does not matter yet, configuration already over-comlicated and useless. So, If someone else agrees to rename I will do.